src/test/java/org/opentripplanner/updater/trip/moduletests/delay/SkippedTest.java

```diff
@@ -6,42 +6,43 @@
 import static org.junit.jupiter.api.Assertions.assertNotSame;
 import static org.junit.jupiter.api.Assertions.assertNull;
 import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.opentripplanner.transit.model._data.TransitModelForTest.id;
 import static org.opentripplanner.updater.spi.UpdateResultAssertions.assertSuccess;
-import static org.opentripplanner.updater.trip.RealtimeTestEnvironment.SERVICE_DATE;
 import static org.opentripplanner.updater.trip.UpdateIncrementality.DIFFERENTIAL;
 
 import org.junit.jupiter.api.Test;
 import org.opentripplanner.transit.model.framework.FeedScopedId;
 import org.opentripplanner.transit.model.timetable.RealTimeState;
 import org.opentripplanner.transit.model.timetable.TripTimesStringBuilder;
+import org.opentripplanner.updater.trip.RealtimeTestConstants;
 import org.opentripplanner.updater.trip.RealtimeTestEnvironment;
 import org.opentripplanner.updater.trip.TripUpdateBuilder;
 
 /**
  * A mixture of delayed and skipped stops should result in both delayed and cancelled stops.
  */
-public class SkippedTest {
+class SkippedTest implements RealtimeTestConstants {
 
   @Test
   void scheduledTripWithSkippedAndScheduled() {
-    var env = RealtimeTestEnvironment.gtfs();
-    String scheduledTripId = env.trip2.getId().getId();
+    var env = RealtimeTestEnvironment.gtfs().withTrip2().build();
```

Comment:
My preference would be to have trip1 and trip2 to be defined in each file that wants them instead of having them defined in the builder. It would be a bit more boilerplate but we would win Locality of Behaviour and looser coupling between the tests. Mainly I don't want to have to jump to another file in order to understand a test.

Reply:
I'm sitting on the fence about this. Lets discuss in the dev meeting.